diff --git a/build/openid4vc-issuer/router/authorizationChallengeEndpoint.mjs b/build/openid4vc-issuer/router/authorizationChallengeEndpoint.mjs
index f4828b39a40180d01e920f8b3fd872e79b9432cb..0901d433ac35ffd746d57803b3b8989a68a5a21f 100644
--- a/build/openid4vc-issuer/router/authorizationChallengeEndpoint.mjs
+++ b/build/openid4vc-issuer/router/authorizationChallengeEndpoint.mjs
@@ -138,8 +138,7 @@ async function handleAuthorizationChallengeNoAuthSession(options) {
 		request,
 		clientAttestation: {
 			...parseResult.clientAttestation,
-			required: issuanceSession.walletAttestation?.required ?? config.walletAttestationsRequired,
-			ensureConfirmationKeyMatchesDpopKey: true
+			required: issuanceSession.walletAttestation?.required ?? config.walletAttestationsRequired
 		},
 		dpop: {
 			...parseResult.dpop,
@@ -215,8 +214,7 @@ async function handleAuthorizationChallengeWithAuthSession(options) {
 		request,
 		clientAttestation: {
 			...parseResult.clientAttestation,
-			required: issuanceSession.walletAttestation?.required,
-			ensureConfirmationKeyMatchesDpopKey: true
+			required: issuanceSession.walletAttestation?.required
 		},
 		dpop: {
 			...parseResult.dpop,
diff --git a/build/openid4vc-issuer/router/pushedAuthorizationRequestEndpoint.mjs b/build/openid4vc-issuer/router/pushedAuthorizationRequestEndpoint.mjs
index 9e8f1dd1a37089e4fd8cc459bac03c8e5c1c5c03..0819329d993fe91ad27594e519a92f80b13a8599 100644
--- a/build/openid4vc-issuer/router/pushedAuthorizationRequestEndpoint.mjs
+++ b/build/openid4vc-issuer/router/pushedAuthorizationRequestEndpoint.mjs
@@ -40,8 +40,7 @@ async function handlePushedAuthorizationRequest(agentContext, options) {
 		request,
 		clientAttestation: {
 			...parsedAuthorizationRequest.clientAttestation,
-			required: issuanceSession.walletAttestation?.required ?? config.walletAttestationsRequired,
-			ensureConfirmationKeyMatchesDpopKey: true
+			required: issuanceSession.walletAttestation?.required ?? config.walletAttestationsRequired
 		},
 		dpop: {
 			...parsedAuthorizationRequest.dpop,
